sql/core/src/test/scala/org/apache/spark/sql/SessionStateSuite.scala

```diff
@@ -39,6 +41,7 @@ class SessionStateSuite extends SparkFunSuite
   protected var activeSession: SparkSession = _
 
   override def beforeAll(): Unit = {
+    doThreadPreAudit()
```

Comment:
Isn't the problem here that this is not calling `super.beforeAll()`? If you do that, you don't need to override `doThreadAuditInSparkFunSuite` nor call `doThreadPostAudit` below.

Reply:
Fixed.